### PR TITLE
Style fix in Company roles

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ n/a
    * stop throbber in case of error and information "no rows" added
    * empty details overlay issue fix by updating the front end business logic of the api key values fetched
 
+* BugFix - Add user role for app not working fix
```
Comment (jjeroch) [inline CHANGELOG.md:22]: @manojava-gk attention - this needs to move into the "unreleased" section

File: cx-portal-shared-components/src/components/basic/Headers/MainHeader/Components/MainHeaderTitle.tsx [hunk 2]
```diff
@@ -27,6 +27,7 @@ export const MainHeaderTitle = ({
   subTitleWidth,
   titleTextVariant = 'h2',
   subTitleTextVariant = 'h2',
```
Comment (jjeroch) [inline cx-portal-shared-components/src/components/basic/Headers/MainHeader/Components/MainHeaderTitle.tsx:29]: @manojava-gk please check this once. As far as I thought those should have 2 different values. One should be H1 and the other one H2 light. Do we have H2 light? If not, we need to get this corrected from the Design side

File: cx-portal-shared-components/src/components/basic/SubNavigation/ParentSubNavigation.tsx [hunk 3]
```diff
@@ -39,8 +39,11 @@ export const ParentSubNavigation = ({
               color="secondary"
               variant="text"
               size="medium"
+              sx={{
+                fontSize: '16px',
+              }}
             >
-              <EastIcon sx={{ marginRight: '16px' }} />
+              <EastIcon sx={{ marginRight: '16px', fontSize: '15px' }} />
```
Comment (jjeroch) [inline cx-portal-shared-components/src/components/basic/SubNavigation/ParentSubNavigation.tsx:46]: `fontsize` why dont we use "body1"?

**Comment by jjeroch** [comment]
@manojava-gk I know you are still working on it, but have added a couple of comments. Can you please plan a localhost review session between you and me for monday. Would like to do this before merging. Thanks!!

File: cx-portal-shared-components/src/components/basic/Headers/MainHeader/Components/MainHeaderTitle.tsx (hunk 2 above)
Comment (manojava-gk) [reply cx-portal-shared-components/src/components/basic/Headers/MainHeader/Components/MainHeaderTitle.tsx:29]: Both title and subtitle was using ```variant="h2"```. Thats why I defined them as ```h2```.

Commit link - https://github.com/catenax-ng/tx-portal-frontend/commit/ccf79c9439dc0cce6473a129fbdb4378d912650a

File: cx-portal-shared-components/src/components/basic/SubNavigation/ParentSubNavigation.tsx (hunk 3 above)
Comment (manojava-gk) [reply cx-portal-shared-components/src/components/basic/SubNavigation/ParentSubNavigation.tsx:46]: ```EastIcon``` does not support ```variant``` props

**Review by oyo — APPROVED** [review]

**Pull request merged**